staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/escaping_test.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
why are these tests not in `validation_test.go`?

Reply:
I've added a bunch of escaping test to `validation_test.go`, removed this `escaping_test.go` file and added a new unit test file for `escaping.go`